includes/Locale.php

```diff
@@ -42,10 +42,26 @@ class Locale {
 	public function get_locale_settings() {
 		global $wp_locale;
 
+		/* translators: Date format, see https://www.php.net/date */
+		$default_date_format = __( 'd/m/Y', 'web-stories' );
+
+		/* translators: Date format, see https://www.php.net/date */
+		$default_time_format = __( 'g:i a', 'web-stories' );
+
+		$date_format = get_option( 'date_format' );
```

Comment:
```suggestion
		$date_format = get_option( 'date_format', $default_date_format );
```

Reply:
That doesn't cover the case where the option is an empty string though, does it?

Reply:
Nope, it doesn't.